tools/accuracy_checker/accuracy_checker/main.py

```diff
@@ -245,6 +245,11 @@ def build_arguments_parser():
         required=False,
         type=str
     )
+    parser.add_argument(
+        '--shuffle', help="Allow shuffle annotation during creation a subset",
+        required=False,
+        type=cast_to_bool
+    )
```

Comment:
this not only way to get annotation, please also update convert.py in annotation_converters

Reply:
New argument is added in convert.py